morphs/data/parse.py

```diff
@@ -17,3 +18,38 @@ def separate_endpoints(stim_id_series):
         stim_ids = stim_ids.str.replace('[a-i]%s128' % (motif), motif)
     end = stim_ids.isin(list('abcdefghi'))
     return stim_ids, end
+
+
+def is_inverted_dim(subj, morph_dim):
+    '''Whether a morph dimension is flipped for the subject'''
+    left, right = morphs.subj.TRAINING[subj].lower().split('|')
+    les, gre = morph_dim
```

Comment:
I'm not sure what les and gre are supposed to represent in relation to morph dimensions. Maybe longer variable names or a comment to explain?